assets/app/views/browse/pod.html

```diff
@@ -96,15 +96,30 @@
                     </select>
 
                     <log-viewer
-                        ng-if="selectedTab.logs"
-                        follow-affix-top="390"
-                        follow-affix-bottom="90"
-                        resource="pods/log"
-                        name="pod.metadata.name"
-                        context="projectContext"
-                        options="logOptions"
-                        time-start="pod.status.startTime | date : 'short'"
-                        run="logCanRun">
+                      ng-if="selectedTab.logs"
+                      follow-affix-top="390"
+                      follow-affix-bottom="90"
+                      resource="pods/log"
+                      name="pod.metadata.name"
+                      context="projectContext"
+                      options="logOptions"
+                      run="logCanRun">
+                      <div row mobile="column" tablet="column">
+                        <div>Container status:&nbsp;</div>
+                        <div row>
+                          <status-icon status="containerExitReason || containerStatusKey"></status-icon>&nbsp;
+                          {{containerExitReason || containerStatusKey | capitalize}}&nbsp;
```

Comment:
Use the `sentenceCase` filter for containerExitReason to be consistent with the browse pods table.